APItokenmanager/Web/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
```

Comment:
nitpick: Why `Web` -- would this sub-directory be more like your API, or perhaps could be named something more descriptive?

Reply:
Web, because, it contains the code for web application, not API. For example, in BrainKB UI (https://github.com/sensein/brainkb-ui/tree/admin-ui/djangoUImanager), there's a case where we have both web application + api and the folders have been renamed accordingly.